djangoRT/templates/djangoRT/ticketCreate.html

```diff
@@ -1,12 +1,37 @@
 {% extends "layout/default.html" %}
-{% load bootstrap3 %}
+{% load bootstrap3 static %}
+
 {% block title %}Create a Ticket{% endblock %}
 
+{% block scripts %}
+{{ block.super }}
+<script src="{% static 'djangoRT/js/question_tree.js' %}"></script>
+{% endblock %}
+
 {% block content %}
 {% include "djangoRT/navigation.html" %}
+<h1>Look up issue</h1>
+<div id="tree-container">
+  <p>
+    You can browse common issues here. Select each answer that fits your
+    problem. If nothing seems to fit, please submit a ticket.
+  </p>
+  <div id="expand_all" class="btn btn-sm btn-primary">Expand All</div>
+  <div id="collapse_all" class="btn btn-sm btn-primary">Collapse All</div>
+  <div id="tree"></div>
```

Comment:
If you're worried about doing things by hand:

```html
{% for group in question_group %}
<details>
    <summary>{{ group.label }}</summary>
    {% if group.desc %}<p>{{ group.desc }}</p>{% endif %}
    {% for question in group.questions %}
    <details>
        <summary>{{ question.label }}</summary>
        <p>{{ question.desc }}</p>
    </details>
    {% endfor %}
</details>
{% endfor %}
```

I think this is arguably better, as it's pretty succinct, and JS could break more easily and doesn't seem to add much value for this UX. You would just need to ensure the 'desc' is treated as a "safe" string by Django so it doesn't escape the link elements.

Reply:
I fully agree, I initially was working on a prototype that used more JS, and so it made sense there.

Reply:
The JS solution also allows for deeper nesting via recursion, but probably is overkill.